setup.py

```diff
@@ -88,15 +53,11 @@ def get_dnsmasq():
       include_package_data=INCLUDE_PACKAGE_DATA, version=VERSION, entry_points=ENTRY_POINTS,
       install_requires=INSTALL_REQUIRES, classifiers=CLASSIFIERS, url=URL, cmdclass=CMDCLASS)
 
-get_dnsmasq()
-
-print()
-print("                     _  __ _       _     _     _               ")
-print("                    (_)/ _(_)     | |   (_)   | |              ")
-print("  ((.))    __      ___| |_ _ _ __ | |__  _ ___| |__   ___ _ __ ")
-print(r"    |      \ \ /\ / / |  _| | '_ \| '_ \| / __| '_ \ / _ \ '__|")
-print(r"   /_\      \ V  V /| | | | | |_) | | | | \__ \ | | |  __/ |   ")
-print(r"  /___\      \_/\_/ |_|_| |_| .__/|_| |_|_|___/_| |_|\___|_|   ")
-print(r" /     \                    | |                                ")
-print("                            |_|                                ")
-print("                                                               ")
+print("Checking required dependencies")
+DEPENDENCIES_RESULT = is_all_dependencies_installed()
```

Comment:
All caps indicates a constant. But this is clearly a variable depending on `is_all_dependencies_installed()` so you should it should be in lowercase.

Reply:
Everything in the **global scope** should be upper case regardless of it's type.
https://github.com/wifiphisher/wifiphisher/blob/f9ae5abd87e1b6d7d966a2a1874249a7b0d082b6/setup.py#L33
Here `find_packages` might return different result based our file structure.

Reply:
The `PACKAGES` also looks wrong to me. 

Quoting PEP8:

`Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL`